_data/retail.yml

```
@@ -418,33 +506,46 @@ websites:
       - phone
       - totp
     doc: https://www.tokopedia.com/panduan-keamanan/fitur-keamanan-tambahan/
+    regions:
+      - id
 
   - name: Trade Me
     url: https://www.trademe.co.nz
     twitter: TradeMe
     facebook: TradeMe
     img: trademe.png
+    regions:
+      - nz
 
   - name: Walgreens
     url: https://www.walgreens.com
     twitter: Walgreens
     img: walgreens.png
+    regions:
+      - us
 
   - name: Walmart
     url: https://www.walmart.com/
     twitter: walmarthelp
     img: walmart.png
+    regions:
+      - us
 
   - name: Warby Parker
     url: https://www.warbyparker.com
     twitter: WarbyParkerHelp
     img: warbyparker.png
+    regions:
+      - ca
+      - us
 
   - name: Wayfair
     url: https://www.wayfair.com
     img: wayfair.png
     twitter: Wayfair
     facebook: wayfair
+    regions:
+      - us
```

```suggestion
    facebook: wayfair
    regions:
      - us
      - ca
      - de
      - uk
```
Wayfair also serves [Canada, Germany, and the United Kingdom](https://www.wayfair.com/international.php).